src/app/series/series.component.html

```diff
@@ -31,6 +31,7 @@ <h1 class="hero-title" *ngIf="!id">Create Series</h1>
     <a routerLinkActive="active" [routerLinkActiveOptions]="{exact:true}" [routerLink]="base">Basic Info</a>
     <a routerLinkActive="active" [routerLink]="[base, 'templates']">Audio Templates</a>
     <a routerLinkActive="active" [routerLink]="[base, 'podcast']">Podcast Distribution</a>
+    <a routerLinkActive="active" [routerLink]="[base, 'feed']">Stories in Feed</a>
```

Comment:
Should probably hide this tab when `series.isNew`.  I'm also leaning towards not using the word "Feed" here, in case this series isn't actually a podcast.  Thoughts?